jwst/extract_2d/grisms.py

```diff
@@ -411,6 +411,7 @@ def extract_grism_objects(input_model,
             # The bounding boxes here are also limited to the size of the detector
             # The check for boxes entirely off the detector is done in create_grism_bbox right now
             y, x = obj.order_bounding[order]
+            log.info(f'YYY, {y}, {clamp(y[0], 0, input_model.meta.subarray.ysize)}')
```

Comment:
Is this the type of info we want appearing in a category `info` message, or is it better suited to `debug` mode?

Reply:
changed